syntax = "proto3";
package position;


enum PosDirection
{
  POS_BUTT = 0;
  POS_LONG = 1;
  POS_SHORT = 2;

}


message AccountPosition
{
  string InstrumentID=1;
  string ProductID = 2;
  PosDirection Direction = 3;
  int64 TotalVolume = 4;
  int64 TodayVolume = 5;
}

message AccPosWrapper
{
  AccountPosition longpos=1;
  AccountPosition shortpos=2;
}